packages/gatsby-cli/src/reporter/reporters/ink/reporter.js

```diff
@@ -126,8 +127,39 @@ export default class GatsbyReporter extends React.Component {
     this._addMessage(`verbose`, str)
   }
 
+  static getDerivedStateFromError(error) {
+    return { hasError: true, error: error.name }
+  }
+
+  componentDidCatch(error, info) {
+    trackBuildError(`INK`, {
```

Comment:
@jamo how do I track this error properly?

Reply:
Yup, this looks correct